modules/machineset-non-guaranteed-instance.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc ?
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc (4.6)
+// * machine_management/creating_machinesets/creating-machineset-gcp.adoc (4.6)
+
+[id="machineset-non-guaranteed-instance_{context}"]
+= MachineSet Spot Instances
+
+You can save costs by creating an AWS MachineSet that deploys machines as non-guaranteed, Spot Instances. Spot Instances utilize unused AWS EC2 capacity and are less expensive than On-Demand Instances. You can use Spot Instances for workloads that can tolerate interruptions, such as batch or stateless, horizontally scalable workloads.
+
+[IMPORTANT]
+====
+It is strongly recommended that control-plane machines are not created on Spot Instances due to the increased likelihood of the instance being terminated. Manual intervention is required to replace a terminated control-plane node.
+====
+
+AWS EC2 can terminate a Spot Instance at any time. AWS gives a two-minute warning to the user when an interruption occurs. The {product-title} begins to remove the workloads from the affected instances when AWS issues this termination notice.
```

Comment:
Perhaps `this termination notice` could be changed to `the termination notice`? You described the warning and then when stating `this termination notice` directly afterwards, it made me initially think you were talking about the warning you just described. I assume that's not the case, and you're talking about after the two-minute warning expires as the termination period?

Reply:
Good observation on the wording. I meant for `warning` and `notice` to be synonymous, but I can see how that is confusing as far as the timing of everything. My understanding is that OCP starts to remove the workloads when AWS issues the two-minute warning/notice. I can change `this termination notice` to `the termination warning` to make that more clear.